presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -53,6 +53,7 @@
 import com.facebook.presto.sql.tree.SortItem;
 import com.facebook.presto.sql.tree.SubqueryExpression;
 import com.facebook.presto.sql.tree.SubscriptExpression;
+import com.facebook.presto.sql.tree.SymbolReference;
```

Comment:
you should also add visiting of `SortItems` in `FunctionCall` in `com.facebook.presto.sql.tree.DefaultTraversalVisitor`